prometheus/go_collector.go

```diff
@@ -245,6 +261,9 @@ func (c *baseGoCollector) Describe(ch chan<- *Desc) {
 	ch <- c.gcDesc
 	ch <- c.gcLastTimeDesc
 	ch <- c.goInfoDesc
+	ch <- c.goMaxProcs
```

Comment:
I wonder if they could fit better in `goCollector` given it already uses runtime metrics and we could give same time snapshot 🤔